ui/components/src/DateTimeRangePicker/TimeRangeSelector.tsx

```diff
@@ -43,6 +43,9 @@ export function TimeRangeSelector(props: TimeRangeSelectorProps) {
       value={formattedValue}
       onChange={onSelectChange}
       IconComponent={Calendar}
+      inputProps={{
```

Comment:
I'm not in love with this, but it is a quick and easy incremental improvement in a11y (and ease of writing tests that use those utils). Previously, it was just the time value without any context.